controllers/items.js

```diff
@@ -1,85 +1,60 @@
-/*jshint esversion: 6 */
+"use strict";
 let fs = require('fs'),
     qs = require('querystring'),
-    config = require('../config');
+    config = require('../config'),
+    db = require('../dbconnect'),
+    getDBRecs =db.getDBRecs,
+    setDBRecs = db.setDBRecs,
+    rmDBRecs = db.rmDBRecs;
 
 exports.getAction = function (request, response) {
-    fs.exists(config.database.path, function (isExists) {
-        response.writeHead(200, {'Content-Type': 'application/json'});
-
-        if (false === isExists) {
-            response.end(JSON.stringify([]));
-
-        } else {
-            fs.createReadStream(config.database.path).pipe(response);
+    getDBRecs((error, data) => {
+        let responseData;
+        if (error) {
+            responseData = error.name + " : " + error.message;
+            response.writeHead(503, {'Content-Type': 'application/json'});
+            response.end(JSON.stringify({responseData: 'Can\'t get data. Please see server\'s console output for details.'}));
         }
+        else {
+            responseData = JSON.stringify(data);
+        }
+        console.log("DB Response: " + responseData);
```

Comment:
You need to put the lines below into the else condition, otherwise it will be executed in case of error